metabase/metabase
# Hide dashcard parameter mappings for action dashboard cards

background: https://metaboat.slack.com/archives/C04FG88HL95/p1677688426691679

### Description

Since we have a snazzy modal for mapping dashboard parameters to actions, we can hide the parameter mapping interface on dashcards, especially given that action dashcards are often very tiny and the mapping component doesn't fit very well.

Before | After
--- | ---
![Screen Shot 2023-03-01 at 10 03 08 PM](https://user-images.githubusercontent.com/30528226/222264436-2a4fc6d6-bdba-471d-b7d2-da486e6b9a83.png) | ![Screen Shot 2023-03-01 at 8 39 15 PM](https://user-images.githubusercontent.com/30528226/222264302-e336a75e-6676-4369-b003-158061af5dce.png)

### How to verify

- Create a dashboard with one or more filters. 
- Add an action with one or more parameters
- While editing the dashboard click on the filters to go into "mapping mode", you should see a nice little info icon instead of a parameter mapping input


### Checklist
